src/components/updatemenus/draw.js

```diff
@@ -299,6 +299,8 @@ function drawButtons(gd, gHeader, gButton, menuOpts) {
             setActive(gd, menuOpts, buttonOpts, gHeader, gButton, buttonIndex);
 
             Plots.executeAPICommand(gd, buttonOpts.method, buttonOpts.args);
+
+            gd.emit('plotly_buttonclicked', {menu: menuOpts, button: buttonOpts, active: menuOpts.active});
```

Comment:
Some people have been asking for an event on legend item click.

I'm not suggesting we should have a lone event for updatemenu button click and legend item click, but we should make sure the event names are consistent.

I'm thinking  `plotly_buttonclicked` and `plotly_legendclick`, unless someone find the two confusing.